functions/http/function.json

```diff
@@ -0,0 +1,97 @@
+{
+  "name": "http",
+  "description": "Sends a HTTP(S) request",
+  "label": "HTTP(S)",
+  "category": "Misc",
```

Comment:
Maybe we can come up with a more specific category. Thinking out loud: External, Communication, Services?

Reply:
I think I prefer `External`. For me this comes the closest to indicating you can use something outside of Betty; in this case doing a http call to something. With Services, I get the idea to have things like `print file` for example.